test/integration/epacems_test.py

```diff
@@ -70,22 +71,14 @@ def test_epacems_subset_input_validation(epacems_year_and_state, epacems_parquet
             epacems(epacems_path=path, **combo)
 
 
-def test_epacems_parallel(pudl_settings_fixture, pudl_ds_kwargs, tmpdir_factory):
+def test_epacems_parallel(pudl_engine):
     """Test that we can run the EPA CEMS ETL in parallel."""
-    epacems_settings = EpaCemsSettings(
-        years=[2019, 2020, 2021], states=["ID", "ME"], partition=True
-    )
     # We need a temporary output directory to avoid dropping the ID/ME 2019/2020
     # parallel outputs in the real output directory and interfering with the normal
     # monolithic outputs.
-    parquet_tmp = tmpdir_factory.mktemp("parquet_tmp")
-    epacems_tmp = Path(parquet_tmp, "epacems")
-    epacems_tmp.mkdir()
-    settings_tmp = {k: v for k, v in pudl_settings_fixture.items()}
-    settings_tmp["parquet_dir"] = parquet_tmp
-    etl_epacems(epacems_settings, settings_tmp, pudl_ds_kwargs)
+    epacems_path = Path(os.getenv("PUDL_OUTPUT")) / "hourly_emissions_epacems"
```

Comment:
Why not use the `epacems_parquet_path` fixture here? Also, does this depend on `pudl_engine` to ensure the cems etl has run?

Reply:
Fixed